core/src/edu/asu/c3simulator/screens/MarketingScreen.java

```diff
@@ -0,0 +1,368 @@
+				0.01f * stage.getHeight());
+		
+		stage.addActor(navigationPanel);
+		stage.addActor(mainTable);
+	}
+	
+	/**
+	 * The method creates the navigation panel.
+	 * 
+	 * @return the NavigationPanel actor that was created.
+	 */
+	private NavigationPanel createNavigationPanel()
+	{
+		NavigationPanel navigationPanel = new NavigationPanel(game, skin);
+		
+		// TODO add screens
+		navigationPanel.addButton("Products", null);
+		navigationPanel.addSubButton("Products", "Pre-Market", null);
+		navigationPanel.addSubButton("Products", "Current", null);
+		navigationPanel.addSubButton("Products", "Retired", null);
+		navigationPanel.addButton("Product Growth", null);
+		navigationPanel.addSubButton("Product Growth", "Supply", null);
+		navigationPanel.addSubButton("Product Growth", "Demand", null);
+		navigationPanel.addButton("Employment", null);
+		navigationPanel.addButton("Marketing", null);
+		
+		navigationPanel.setPosition(0.01f * stage.getWidth(), stage.getHeight()
+				- (0.3f * stage.getHeight()));
+		
+		return navigationPanel;
+	}
+	
+	/**
+	 * This method creates the main section of the GUI
+	 * 
+	 * @return The table that was created.
+	 */
+	private Table createMainTable()
+	{
+		Table mainTable = new Table();
+		mainTable.setSize(0.80f * stage.getWidth(), 0.80f * stage.getHeight());
+		
+		Table leftSection = new Table();
+		Table rightSection = new Table();
+		
+		ScrollPane productGridScroll = new ScrollPane(getProductGrid(), skin);
+		companyPromotionButton = new TextButton("Promote Company", skin);
+		companyPromotionButton.addListener(new ClickListener() {
+			@Override
+			public void clicked(InputEvent event, float x, float y)
+			{
+				if (promoteCompany)
+					companyPromotionButton.setColor(1.0f, 1.0f, 1.0f, 1.0f);
+				else
+					companyPromotionButton.setColor(0.0f, 1.0f, 0.0f, 1.0f);
+				promoteCompany = !promoteCompany;
+				productChosen = false;
+				if (selectedProduct != null)
+					selectedProduct = null;
+				if (selectedProductLabel != null)
+				{
+					selectedProductLabel.setColor(1.0f, 1.0f, 1.0f, 1.0f);
+					selectedProductLabel = null;
+				}
+			}
+		});
+		
+		leftSection.add(productGridScroll).expand().fill().row();
+		leftSection.add(companyPromotionButton).center().space(10.0f);
+		
+		ScrollPane marketingGridScroll = new ScrollPane(getMarketingGrid(), skin);
+		TextButton marketinLabel = new TextButton("Select Marketing", skin);
+		marketinLabel.setDisabled(true);
+		marketinLabel.setColor(0.5f, 0.5f, 0.5f, 1.0f);
+		TextButton nextButton = new TextButton("Next", skin);
+		nextButton.addListener(new ClickListener() {
+			@Override
+			public void clicked(InputEvent event, float x, float y)
+			{
+				if (marketingMethodChosen && (productChosen || promoteCompany))
+				{
+					System.out.println("Moving to the next Step. "
+							+ selectedMarketingMethod
+							+ " will be used to market "
+							+ (productChosen ? selectedProduct.getName() : "the Company."));
```

Comment:
All of these println statements will need to be replaced with logger statements eventually.